testdata/scripts/first.txt

```diff
@@ -0,0 +1,10 @@
+exec devnet &
+
+exec sleep 20
+
+env TOKEN=`cat ~/.devLotus/token`
+env API=`cat ~/.devLotus/api`
+env DATADIR=/tmp
+env MINER=t01000
+
+exec dealbot --api $TOKEN:$API storage-deal --data-dir $DATADIR --miner $MINER
```

Comment:
you should add dealbot as a TestMain command, like so: https://github.com/burrowers/garble/blob/3afc993266c3f331bc116eeea0fd224a266c64de/main_test.go#L33

Then here you can `exec dealbot` or just `dealbot`, and it will run the binary magically without you having to `go install` it into the global PATH previously.